visualization/include/pcl/visualization/area_picking_event.h

```diff
@@ -64,13 +69,40 @@ namespace pcl
         {
           if (nb_points_ <= 0)
             return (false);
-          indices = indices_;
+          for (const auto& i : cloud_indices_)
+            indices.insert(indices.cend (), i.second.cbegin (), i.second.cend ());
           return (true);
         }
+        /** \brief For situations where a whole area is selected, return the names
+          * of the selected point clouds.
+          * \return The names of selected point clouds
+          */
+        inline std::vector<std::string>
+        getCloudNames () const
+        {
+          std::vector<std::string> names;
+          for (const auto& i : cloud_indices_)
+            names.push_back (i.first);
+          return names;
+        }
+        /** \brief For situations where a whole area is selected, return the points indices
+          * for a given point cloud
+          * \param[in] name of selected clouds.
+          * \return The indices for the selected cloud.
+          */
+        inline Indices
+        getCloudIndices (const std::string& name) const
```

Comment:
I think `getCloudIndices()` is somewhat misleading, I'd expect to get indices of clouds, whatever that might be. Why don't we stick with existing naming and call this `getPointsIndices(const std::string& name)`?

Reply:
agree. Change name to getPointsindices